src/index.js

```diff
@@ -1,5 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { rehydratedStorePromise } from './_store/configureStore';
+import * as LiveData from './_data/LiveData';
+import { tryAuth } from './_data/Auth';
 import Root from './_store/root';
 
-ReactDOM.render(<Root/>, document.getElementById('root'));
+rehydratedStorePromise.then(st => {
```

Comment:
That is something goes wrong with the call, whole app is down. Just an observation ;)